ui/js/selectors/app.js

```diff
@@ -39,34 +39,37 @@ export const selectPageTitle = createSelector(
   (page, params) => {
     switch (page) {
       case 'search':
-        return params.query ? `Search results for ${params.query}` : 'Search'
+        return params.query ? __("Search results for %s", params.query) : __('Search')
       case 'settings':
-        return 'Settings'
+        return __('Settings')
       case 'help':
-        return 'Help'
+        return __('Help')
       case 'report':
-        return 'Report'
+        return __('Report')
       case 'wallet':
+        return __("Wallet")
       case 'send':
+        return __("Send")
       case 'receive':
+        return __("Receive")
       case 'rewards':
-        return page.charAt(0).toUpperCase() + page.slice(1)
```

Comment:
This code was correct. Yes, it means it cannot easily be translated. For tricky ones like this, just make a note and we'll figure out how to resolve them.